step-release-vis/src/app/services/coloringService_test.ts

```diff
@@ -228,4 +228,22 @@ describe('ColoringService', () => {
       expect(service.edgeOccurrences.size).toEqual(0);
     });
   });
+
+  describe('#getColorFromIndex', () => {
```

Comment:
I would expect many more cornercases to be present here, let's have at least 1 test with at least 5 candidates ;)